README.md

```diff
@@ -14,12 +13,10 @@ products:
 # Group Chat Sample
 
 This is a sample application to show how the Chat Web SDK can be used to build a single threaded chat experience.
-The client-side application is a React based user interface which uses Redux for handling complex state while leveraging Microsoft Fluent UI.
-Powering this front-end is a C# web application powered by ASP.NET Core to connect this application with Azure Communication Services.
 
 Additional documentation for this sample can be found on [Microsoft Docs](https://docs.microsoft.com/en-us/azure/communication-services/samples/chat-hero-sample).
 
-![Homepage](./Media/homepage-sample-chat.png)
+![Homepage](./Chat/Media/homepage-sample-chat.png)
```

Comment:
The Chat folder that everything is in seems unnecessary - this repo only has `Chat/` things in it